### PR TITLE
🤩 Added custom openai tts models and role variables

### PR DESCRIPTION
**Pull request by Yanyutin753**
## Description

🤩 Added custom openai tts models and role environment variables

#1978 

| Variable Name | Description |
| --- | --- |
| `AUDIO_OPENAI_API_MODEL` | This variable is used to select which OpenAI Text to Speech (TTS) model to use. `tts-1-hd` and `tts-1` |
| `AUDIO_OPENAI_API_VOICE` | This variable is used to select which OpenAI synthetic voice to use. The OpenAI API offers a variety of voices, including different genders and accents from different regions, to cater to various use cases. |

---

### Added

 `AUDIO_OPENAI_API_MODEL`  and `AUDIO_OPENAI_API_SPEAKER`

---

### Additional Information

Self test passed

image

image

### CODE REVIEW

File: backend/apps/audio/main.py [hunk 1]
```diff
@@ -43,6 +43,8 @@
     DEVICE_TYPE,
     AUDIO_OPENAI_API_BASE_URL,
     AUDIO_OPENAI_API_KEY,
+    AUDIO_OPENAI_API_MODEL,
+    AUDIO_OPENAI_API_SPEAKER,
```
Comment (tjbck) [inline backend/apps/audio/main.py:47]: Should be renamed to `AUDIO_OPENAI_API_VOICE`

File: src/lib/apis/audio/index.ts [hunk 2]
```diff
@@ -95,7 +97,8 @@ export const transcribeAudio = async (token: string, file: File) => {
 export const synthesizeOpenAISpeech = async (
 	token: string = '',
 	speaker: string = 'alloy',
-	text: string = ''
+	text: string = '',
+	OpenAIModel: string = 'tts-1'
```
Comment (tjbck) [inline src/lib/apis/audio/index.ts:101]: `model` would suffice here

File: backend/apps/audio/main.py (hunk 1 above)
Comment (Yanyutin753) [reply backend/apps/audio/main.py]: ok

File: src/lib/apis/audio/index.ts (hunk 2 above)
Comment (Yanyutin753) [reply src/lib/apis/audio/index.ts]: yes, i have modified it

**Comment by Yanyutin753** [comment]
@tjbck Let's see if it works now

**Comment by tjbck** [comment]
LGTM, Thanks!

**Comment by Yanyutin753** [comment]
🥳

**Pull request merged**